htdocs/js/smr15.js

```diff
@@ -102,8 +102,57 @@ function showRaceInfo(select) {
 	var desc = document.getElementById('race_descr');
 	$(desc).children().addClass('hide');
 	$(".race_descr" + race_id, desc).removeClass('hide');
+	graph_by_plotly(race_id);
 }
 
+// Used by game_join.php 
+function graph_by_plotly(race_id) {
+
+	var races = [["[1.0,1.0,1.0,1.0,1.0]","'Neutral'","'#FFD800'"],
```

Comment:
A few minor issues with the `races` data structure:
1. Formatting
    * Add spaces between items (i.e. between the values, the race name, and the color).
    * I don't know what the name of the style is, but for consistency let's format arrays like
```
array = [
   [[1.0,1.0,1.0,1.0,1.0], 'Neutral', '#FFD800'],
   ...
];
```
2. There is a typo with `Alskant`. I don't like duplicating the names and colors here, because duplication leads to human error (typos, desync, etc.). While it would be pretty easy to pass the race names in from PHP, the colors would be challenging, because they only exist in the CSS files, and I don't know of a simple way to generate the CSS files from PHP with our infrastructure.
3. Dummy 0th index should be removed. I know it's annoying that the races are 1-indexed instead of 0-indexed. Perhaps it would be better to use an associative array, and then the keys can be the actual `race_id` without a dummy first entry, e.g.
```
var races = {
  1: [[1.0,1.0,1.0,1.0,1.0], 'Neutral', '#FFD800'],
  2: ...
};
```